ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -11040,11 +11040,21 @@ keys:
               description: Tcam-Profile Name
             config:
               type: str
-              required: true
+              required: false
               description: 'TCAM Profile Config. Since these can be very long, it
                 is often a good idea to import the config from a file.
 
-                Example: "{{ lookup(''file'', ''TCAM_TRAFFIC_POLICY.conf'') }}"'
+                Example: "{{ lookup(''file'', ''TCAM_TRAFFIC_POLICY.conf'') }}"
+
+                '
+            source:
+              type: str
+              required: false
+              description: 'TCAM profile local source path. Used to read the TCAM
+                profile from a local path existing in the device.
+
+                The file can contain different TCAM profiles and the one used will
+                be defined by the "system" variable.'
```

Comment:
Are we sure about this? The file is given under one profile, so how can it contain multiple profiles?